packages/DatePicker/stories/screener.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import moment from "moment";
+import { storiesOf } from "@storybook/react";
+import L10n from "@paprika/l10n";
+import DatePicker from "../src/DatePicker";
+import ShortCutPanel from "../src/components/ShortcutPanel";
+
+const noop = () => {};
+
+storiesOf("DatePicker / screener", module)
+  .add("Calendar", () => (
+    <L10n>
+      <DatePicker.Calendar date={moment("2019-01-01", "YYYY-MM-DD")} isOpen onSelect={noop} />
+    </L10n>
+  ))
+  .add("Shortcut panel", () => (
+    <L10n>
+      <ShortCutPanel date={moment("2019-01-01", "YYYY-MM-DD")} onCancel={noop} onConfirm={noop} />
```

Comment:
Can you add an image of what is ShortCutPanel? is part of the DatePicker?  might be a better developer experience in the sense of API for using the component?